YesterdayApi/Web/Cors/CorsExtension.cs

```diff
@@ -0,0 +1,18 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace YesterdayApi.Web.Cors
+{
+  public static class CorsExtension
+  {
+    public static void AddCorsPolicy(this IServiceCollection services)
+    {
+      services.AddCors();
+    }
+
+    public static void UseCorsPolicy(this IApplicationBuilder app)
+    {
+      app.UseCors(builder => builder.AllowAnyOrigin());
```

Comment:
I'm not sure is this enough to allow us to use GET/POST/PUT/PATCH and pass Header for credentials but we gonna see that. If yes somebody will fix that.